bounded-collections/src/lib.rs

```diff
@@ -51,6 +51,38 @@ impl<T: Default> Get<T> for () {
 	}
 }
 
+/// Converts [`Get<I>`] to [`Get<R>`] using [`Into`].
+///
+/// Acts as a type-safe bridge between `Get` implementations where `I: Into<R>`.
+///
+/// - `Inner`: The [`Get<I>`] implementation
+/// - `I`: Source type to convert from
+/// - `R`: Target type to convert into
+///
+/// # Example
+/// ```
+/// use bounded_collections::Get;
+/// use bounded_collections::GetInto;
+///
+/// struct MyGetter;
+/// impl Get<u16> for MyGetter { fn get() -> u16 { 42 } }
+/// assert_eq!(GetInto::<MyGetter, u16, u32>::get(), 42u32);
+/// ```
+pub struct GetInto<Inner, I, R>(core::marker::PhantomData<(Inner, I, R)>);
```

Comment:
```suggestion
pub struct GetInto<Inner, I>(core::marker::PhantomData<(Inner, I)>);
```

This should work as well and would make the type easier to use.

Reply:
Updated